src/nfts/selectors.ts

```diff
@@ -0,0 +1,7 @@
+import { RootState } from 'src/redux/reducers'
+
+export const nftsLoadingSelector = (state: RootState) => state.nfts.nftsLoading
+
+export const nftsErrorSelector = (state: RootState) => state.nfts.nftsError
```

Comment:
nit: we started doing this in the positions stuff, but it seems like it'd be nice to have a single state to represent status (e.g. status = 'loading' | 'error' | 'success' etc) to prevent the loading and error states from being simultaneously true. it also makes rendering a little easier. idk if you want to do that here, but wanted to call it out. changing the state would require a redux migration but you could also just derive this on the selector level

Reply:
A follow up PR with a redux migration and adjusting this component seems like a good place to make this change. I agree having a single state to represent the status of the component makes a lot more sense to me.